plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -72,11 +80,51 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 		};
 	}, [ currentFields, currentFieldConfig, currentCountry, addressType ] );
 
+	const address2Field = addressFormFields.fields.find(
+		( field ) => field.key === 'address_2'
+	);
+
+	const hasAddress2FieldValue = useCallback( () => {
+		return objectHasProp( values, 'address_2' ) && values.address_2 !== '';
+	}, [ values ] );
```

Comment:
Does this need to be a function? seems like a memo would be good here.

Reply:
Good catch. I changed it into `useMemo()`.